i18n/en.json

```diff
@@ -564,8 +564,32 @@
     "translation": "Error to update the current channel."
   },
   {
-    "id": "api.command_channel_header.update_channel.success",
-    "translation": "Channel header successfully updated."
+    "id": "api.command_channel_purpose.name",
+    "translation": "purpose"
+  },
+  {
+    "id": "api.command_channel_purpose.hint",
+    "translation": "[text]"
+  },
+  {
+    "id": "api.command_channel_purpose.desc",
+    "translation": "Edit the channel purpose."
+  },
+  {
+    "id": "api.command_channel_purpose.channel.app_error",
+    "translation": "Error to retrieve the current channel."
+  },
+  {
+    "id": "api.command_channel_purpose.permission.app_error",
+    "translation": "Only [Channel, Team and System Admins / Team and System Admins / System Admins] are allowed to edit the channel purpose."
```

Comment:
Can we change this text to `You do not have the appropriate permissions to edit the channel header.`